python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -17,14 +18,31 @@
 )
 
 
-def insert_trips_hash_columns(
+def load_temp_for_hash_compare(
     db_manager: DatabaseManager, events: pandas.DataFrame
 ) -> None:
     """
-    Load new 'trip_hash' records and columns into RDS
+    Load "temp_hash_compare" table with "trip_hash" from newly inserted RT events
+    """
+    new_trip_hash_bytes = events.loc[
+        events["do_insert"], "trip_hash"
+    ].drop_duplicates()
+
+    db_manager.truncate_table(TempHashCompare)
+    db_manager.execute_with_data(
+        sa.insert(TempHashCompare.__table__),
+        new_trip_hash_bytes.to_frame("trip_stop_hash"),
+    )
+
+
+def load_new_trip_data(
+    db_manager: DatabaseManager, events: pandas.DataFrame
+) -> None:
+    """
```

Comment:
This works the same as before, just with slightly less scope in this function and the addition of `trip_id`